src/software/ai/hl/stp/action/intercept_ball_action.h

```diff
@@ -17,7 +17,7 @@ class InterceptBallAction : public Action
      * @param loop_forever Continue yielding new Move Intents, even after we have reached
```

Comment:
update javadoc comment to remove `loop_forever` (not going to comment on all of them so please check all the actions)

Reply:
went through all actions and updated javadoc